betting-game-react/src/components/AboutUs/AboutUs.test.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import AboutUs from './AboutUs';
+
+describe('<AboutUs />', () => {
```

Comment:
Is this actually a best practice for "describe"? I don't remember seeing that yet ~
Maybe it'd be better to use the describe name for a domain explanation? 
E.g: 
`describe('Component that shows information about our company', () => { ...`